samples/bluetooth/mesh/light_dimmer/README.rst

```diff
@@ -155,13 +164,15 @@ Configure the Generic Level Client model on the first element on the **Mesh Ligh
 
 Configure the first element on the target **Mesh Light Fixture** node:
 
-* Bind the following models to **Application Key 1**: Generic Level Server and Generic OnOff Server.
-* Subscribe both models to the same group as set for publication for the **Mesh Light Dimmer**.
+* Bind the following models to **Application Key 1**: Generic Level Server, Generic OnOff Server, Scene Server, and Scene Setup Server.
+* Subscribe all models to the same group as set for publication for the **Mesh Light Dimmer**.
 
 You should now be able to perform the following actions:
 
 * Press and release **Button 1** to see the LED on the target Mesh Light Fixture device toggle on and off.
 * Press and hold **Button 1** to see the light level of the LED on the target Mesh Light Fixture device slowly decrease or increase.
+* Short press **Button 2** to rotate through scenes, recalling each of them in turn.
+* Long press **Button 2** to store the current LED states as a scene with the current scene number.
```

Comment:
```suggestion
* Press and hold **Button 2** to store the current LED states as a scene with the current scene number.
```

Reply:
I think we should at least align this the language between the two buttons. Does long press also include release, or is it press and hold. (We should explain short and long press)
```suggestion
* Press and hold **Button 2**, on release the current LED states will be stored with the current scene number.
```